### PR TITLE
Clarify doc for deoplete#mappings#manual_complete

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: doc/deoplete.txt [hunk 1]
```diff
@@ -267,16 +267,20 @@ KEY MAPPINGS 					*deoplete-key-mappings*
...
-		If you give {sources} argument, deoplete will call {sources}.
-		{sources} is name of source or list of sources name.
+		Use this function with |:inoremap| <expr> (|:map-expression|).
+		It calls the completion of deoplete.  You can use it with
+		custom completion setups.
+		You can provide a list of {sources}: It can be the name of a
+		source or a list of sources name.
 >
-		inoremap <silent><expr><Tab>
+		See also |g:deoplete#disable_auto_complete|, which should be
```
Comment (Shougo) [inline doc/deoplete.txt]: Why this description is added?
Please describe it to me.

Comment (blueyed) [inline doc/deoplete.txt]: You mean the reference to `g:deoplete#disable_auto_complete`?

It should be used if you want to trigger the completion manually, no?

Comment (Shougo) [inline doc/deoplete.txt]: OK.  Please add the description for it.

```
If you want to use deoplete as manual completion plugin, see also |g:deoplete#disable_auto_complete|, which should be 1 then typically.
```

**Comment by Shougo** [comment]
@blueyed Ping

**Comment by Shougo** [comment]
@blueyed Can you fix it?

**Comment by blueyed** [comment]
Amended (slightly reworded).

**Comment by Shougo** [comment]
OK, thanks.

**Comment by blueyed** [comment]
You were too fast.  I was about to force-push a fix - the "code selection marker" is wrong.

**Comment by blueyed** [comment]
See https://github.com/Shougo/deoplete.nvim/pull/86/files.

**Pull request merged**